packages/wxt/src/client/content-scripts/ui/index.ts

```diff
@@ -27,16 +34,38 @@ export function createIntegratedUi<TMounted>(
   wrapper.setAttribute('data-wxt-integrated', '');
 
   let mounted: TMounted | undefined = undefined;
+  let autoMountInstance: AutoMount | undefined = undefined;
   const mount = () => {
     applyPosition(wrapper, undefined, options);
     mountUi(wrapper, options);
     mounted = options.onMount?.(wrapper);
   };
+  const autoMount = (autoMountOptions?: AutoMountOptions) => {
+    if (autoMountInstance) {
+      throw Error('autoMount is already set.');
+    }
```

Comment:
Should we throw an error here, or just return and do nothing?

Reply:
@aklinker1 Hmm, but If don't throw it, the return value to be `StopAutoMount | undefined`. Or maybe warn logger is better.

Reply:
Yeah, I like this better.